### DISCUSSION TITLE
Logo

### DISCUSSION THREAD

**Discussion by NicolasCARPi** [TheELNConsortium/TheELNFileFormat#46] [General]
We need a logo.

![logo-color](https://user-images.githubusercontent.com/3043706/192118195-1fa418ee-48aa-47c2-9598-7097f552e735.png)

## Have fun with https://app.logo.com/ or a graphics editor if you have the skills (I don't) and propose yours!

All the files:
[logo-elnconsortium-nico_0.zip](https://github.com/TheELNConsortium/Discussions-And-Future-Topics/files/9639551/logo-elnconsortium-nico_0.zip)

**Comment by SteffenBrinckmann** [comment]
I like it and the color
